### DISCUSSION TITLE
Добавление своей музыки на ЯМу и открытие её для всех

### DISCUSSION THREAD

**Discussion by P4EJIA** [Chimildic/YaMuTools#73] [Идея] [answered]
Здравствуйте
Как известно, Яндекс добавил возможность раскрывать залитые треки для всех, обходя отсутствие лицензий правообладателей. 
Добавьте возможность автоматически маркировать все треки в плейлисте на открытый доступ. В текущем виде нужно каждый отдельный трек вручную прокликивать. 
https://i.vgy.me/T9p4T8.jpg
https://i.vgy.me/rGHjar.jpg

**Comment by Chimildic** [comment, marked as answer]
Сделал в версии 0.8.8. Обновление на проверке в сторе.

**Reply by kisoid** [reply]
https://habr.com/ru/news/t/706212/

> В пресс-службе «Яндекса» Хабру рассказали, что пользователи, вероятно, столкнулись с багом. Возможность делиться загруженными треками не связали с ноябрьским анонсом. Также пресс-служба пообещала разобраться с произошедшим и рассказать об итогах.

**Reply by P4EJIA** [reply]
Главное что возможность есть и она работает, а называть они это могут как угодно. 
Еще раз спасибо за оперативное обновление

**Reply by Chimildic** [reply]
Будьте осторожны. В версии 0.8.8 баг с автосозданием плейлистов, если зайти в лайки. Фикс сделал, но гугл стор не пропускает его пока.

**Comment by AlexByte** [comment]
Почему у меня пишет "Данный трек нельзя опубликовать"?

**Reply by kisoid** [reply]
Прикрыли лавочку, значит.
